src/elements/helper/CssUnitConverter.ts

```diff
@@ -57,6 +57,8 @@ export function convertCssUnitToPixel(cssValue: string, target: HTMLElement, per
 }
 
 export function getCssUnit(cssValue: string) {
+    if(cssValue.substring(cssValue.length - 2, cssValue.length) == "fr")
+        return "fr";

```

Comment:
mach doch einfach fr oben in die units liste

Reply:
erledigt